### PR TITLE
Adding a deploy script for WMCore services.

### PR DESCRIPTION
**Pull request by arooshap**
Hi,

In order to deploy the services in the dmwm and tzero namespace, I am adding this script. I have tested it in the test7 cluster, and it works. 

Thanks

### CODE REVIEW

**Comment by muhammadimranfarooqi** [comment]
Hi @amaltaro 

This also implies that the permissions should be granted to tzero guys in dmwm namespace?

**Comment by amaltaro** [comment]
>This also implies that the permissions should be granted to tzero guys in dmwm namespace?

I don't think they have ever done anything with the t0_reqmon service, other than looking at the logs. So, I'd say they do not need to have access to the `dmwm` namespace.

**Review by muhammadimranfarooqi — COMMENTED** [review]
Hi @arooshap 

The `test12` cluster is missing. Please add it as well

File: kubernetes/cmsweb/scripts/deploy-srv-wmcore.yaml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        if [ -z "`grep imagetag $i.yaml`" ]; then
+         echo $i
+         echo "unable to locate imagetag in $i.yaml"
+        exit 1
+        fi
+        echo "The downloaded and newly generated service manifest files are available at $tmpDir"
+        echo "Using Environment: $cmsweb_env"
+
+        # replace imagetag with real value and deploy new service
+        if [ "$cmsweb_env" == "k8s-prod" ] ; then
```
Comment (muhammadimranfarooqi) [inline kubernetes/cmsweb/scripts/deploy-srv-wmcore.yaml:174]: @arooshap 

Please add OR condition of `k8s-preprod` as well as #PROD# prefix is replaced in production and testbed clusters. You also need to remove the check from below regarding `k8s-preprod`

**Comment by arooshap** [comment]
After the migration of t0reqmon to `dmwm` namespace, I have made the recommended changes to the script.

@muhammadimranfarooqi can you please review it?

**Pull request merged**